.gitignore

```diff
@@ -19,3 +19,13 @@ content/docs/reference/components/**/*.md
 content/docs/guides/amp-cors-requests.md
 content/docs/guides/amp-cache-debugging.md
 content/support/developer/get_support.md
+
+content/docs/contribute/governance.md
+content/docs/fundamentals/spec.md
+content/docs/fundamentals/spec/*.md
+content/docs/fundamentals/amp-cors-requests.md
+content/docs/design/amp-html-layout.md
+content/docs/interaction_dynamic/amp-actions-and-events.md
```

Comment:
some of the things in here doesn't seem like generated things..or am I wrong?

Reply:
they will be. :)